src/containers/Home.js

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect, useState } from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import {
+  changePage,
+  deleteItem,
+  getArticleList,
+  getNewsItems,
+  markasRead,
+  setArticles,
+} from "../actions/action";
+import { getArticles } from "../utils/apis";
+import Article from "./Article";
+import Paginator from "./Paginator";
+
+const Home = ({
+  list,
+  page,
+  getNewsItems,
+  getArticleList,
+  setArticles,
+  isFetching,
+  articles,
+  deleteItem,
+  deletedList,
+  markasRead,
+  changePage,
+}) => {
+  const [articlesList, setarticlesList] = useState([]);
+
+  useEffect(() => {
+    getNewsItems();
+  }, []);
+
+  useEffect(() => {
+    getArticleList();
+    getArticles(list, page)
+      .then((stories) => {
+        setarticlesList(stories);
+      })
+      .catch(() => {});
```

Comment:
This should be caught in redux

Reply:
moved the getArticles call and catch to redux actions